app/routes/project/conversations/index.js

```diff
@@ -3,10 +3,14 @@ import Route from '@ember/routing/route';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 
 export default Route.extend(AuthenticatedRouteMixin, {
-  redirect() {
+  redirect(model, transition) {
+    let status = transition.queryParams.status || 'open';
     let { conversations } = this.modelFor('project.conversations');
-    if (get(conversations, 'length') > 0) {
-      this.transitionTo('project.conversations.conversation', get(conversations, 'firstObject'));
+    let sortedConversations
+      = conversations.filterBy('status', status).sortBy('updatedAt');
```

Comment:
I don't think you need to filter here, right? The model should already be filtered. If it's not, then we should add a comment as to why that is.

You do need to sort, but not until you know the array isn't empty. That shouldn't matter much, though. Sorting a blank array is not expensive.

Reply:
The filtering does need to happen since the model does not necessarily update when an individual conversation changes.